WebInstall/explainer_cross_domain.md

```diff
@@ -101,10 +121,10 @@ The `navigator.install` call can receive an object with a set of parameters that
 To install a same domain web site/app, the process is as follows:
 1. Origin site that triggers the installation must have installation permissions as it tries to install a cross-origin app.
 2. target site/app must comply with *[installability criteria](#installability-criteria), if any*.
```

Comment:
Is this still true for same domain?

Reply:
Nvm, this is for the cross domain explainer. Can you update the text above to:
"To install a *cross* domain web site/app, the process is as follows:"

Reply:
updated the text